.env.reference

```
@@ -0,0 +1,451 @@
+# Allowed origins for CORS.
+# WARNING: Default is wildcard "*" — allows ANY website to make cross-origin
+# requests.  For financial services, ALWAYS set specific trusted origins.
+# Comma-separated for multiple origins.
+# Type: string (comma-separated origins)
+# Default: "*"
+ACCESS_CONTROL_ALLOW_ORIGIN=*
+
+# Allowed HTTP methods for CORS preflight.
+# Type: string (comma-separated methods)
+# Default: "POST, GET, OPTIONS, PUT, DELETE, PATCH"
+ACCESS_CONTROL_ALLOW_METHODS=POST, GET, OPTIONS, PUT, DELETE, PATCH
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)